tests/wgsl-errors.rs

```diff
@@ -13,6 +13,7 @@ fn check(input: &str, snapshot: &str) {
                 diff::Result::Right(r) => println!("+{}", r),
             }
         }
+        println!("Raw output:\n{}", output);
```

Comment:
debug stuff?

Reply:
Yep, easier copy-paste for tests actual\expected stuff.